services/redis.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: sd-redis
+  labels:
+    name: redis
```

Comment:
Not sure if we want the `name: redis` in label. Cuz the name of the service is `sd-redis`. Why do we want to select the pod with label `name:redis`?